### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by moodmosaic**
Some housekeeping I did while going through the new Fable test-suite. Feel free to discuss and/or reject any or or all of those:

* remove redundant space
* remove unused `open` directives
* `testCaseNoFable` renamed to `fableIgnore`
* where possible maximum line width at 80 characters
* where possible use `expected`/`actual` test terminology

/cc @adam-becker @TysonMN

### CODE REVIEW

File: tests/Hedgehog.Tests/TestDsl.fs
Comment (moodmosaic) [inline tests/Hedgehog.Tests/TestDsl.fs:37]: I'm not sure off the top of my head what's the IL diff between inlining vs no-inlining those. (I think, last time I had to deal with IL was [in 2012](https://github.com/AutoFixture/SemanticComparison/commit/af632fc95297b6bf1fa30a5cc0bf42472e46b773#diff-cb5da29d5d5e182dadee4df2aa3f982f4dc48c161e001feaf9045e6fd6baa79d) :open_mouth:, or [is it similar to GHC's optimizations](https://downloads.haskell.org/~ghc/8.6.3/docs/html/users_guide/glasgow_exts.html#inline-pragma)?)

Comment (adam-becker) [reply tests/Hedgehog.Tests/TestDsl.fs:37]: I'm completely in the dark there too, `inline` makes perfect sense to me when compiling, but how a compiled form retains the ability to be inlined is beyond me.

Comment (moodmosaic) [reply tests/Hedgehog.Tests/TestDsl.fs:37]: Perhaps we can take those out. (Less is more(?))

File: tests/Hedgehog.Tests/GenTests.fs
Comment (adam-becker) [inline tests/Hedgehog.Tests/GenTests.fs:9]: I'd like to see if we can get rid of the `<|` in these tests, only because we decided to do it in #276.

File: tests/Hedgehog.Tests/TestDsl.fs
Comment (adam-becker) [reply tests/Hedgehog.Tests/TestDsl.fs:37]: Since this test project isn't a library, I think inline here doesn't hurt anything. Usually `inline` is used to mitigate overhead, and to do fun ad-hoc things like we have in `Numeric.fs`. We don't really need either in this case, so it's up to you @moodmosaic.

File: tests/Hedgehog.Tests/GenTests.fs
Comment (adam-becker) [inline tests/Hedgehog.Tests/GenTests.fs:70]: Do we want to always assume `\n` here? Does this break Windows users?

Comment (moodmosaic) [reply tests/Hedgehog.Tests/GenTests.fs:70]: No, I made a mistake here. I'll revert that one. Very good catch :+1:

File: tests/Hedgehog.Tests/GenTests.fs
Comment (moodmosaic) [reply tests/Hedgehog.Tests/GenTests.fs:9]: Yes, I think so too. Let's do this on a separate PR.

File: tests/Hedgehog.Tests/TestDsl.fs
Comment (moodmosaic) [reply tests/Hedgehog.Tests/TestDsl.fs]: Let's remove it. I just forced-pushed on the branch.

File: tests/Hedgehog.Tests/GenTests.fs
Comment (moodmosaic) [reply tests/Hedgehog.Tests/GenTests.fs]: Done.

**Comment by moodmosaic** [comment]
I just force-pushed the branch from 667f37d to d694831. Thanks for the feedback @adam-becker :+1:

File: tests/Hedgehog.Tests/GenTests.fs
Comment (adam-becker) [inline tests/Hedgehog.Tests/GenTests.fs:64]: I think we can open `System` and not fully qualify `System.DateTime`. If `DateTime` is going to collide, we can use a type alias.

File: tests/Hedgehog.Tests/GenTests.fs
Comment (adam-becker) [inline tests/Hedgehog.Tests/GenTests.fs:74]: This value is repeated a bit, can we make a binding for it?

File: tests/Hedgehog.Tests/SeedTests.fs
Comment (adam-becker) [inline tests/Hedgehog.Tests/SeedTests.fs:27]:
```fsharp
let actual =
  { Value = value
    Gamma = gamma }

actual =! Seed.from x
```

File: tests/Hedgehog.Tests/GenTests.fs
Comment (adam-becker) [inline tests/Hedgehog.Tests/GenTests.fs:47]: The parens aren't needed here.

**Review by adam-becker — APPROVED** [review]
LGTM, left a few more comments, but none of them are blocking.

File: tests/Hedgehog.Tests/GenTests.fs
Comment (moodmosaic) [reply tests/Hedgehog.Tests/GenTests.fs]: :+1:

File: tests/Hedgehog.Tests/SeedTests.fs
Comment (moodmosaic) [reply tests/Hedgehog.Tests/SeedTests.fs]: :+1: (Space [isn't needed](https://blog.ploeh.dk/2013/06/24/a-heuristic-for-formatting-code-according-to-the-aaa-pattern/) there.)

File: tests/Hedgehog.Tests/GenTests.fs
Comment (moodmosaic) [reply tests/Hedgehog.Tests/GenTests.fs]: It's just in 2 places so I left it as-is.

File: tests/Hedgehog.Tests/GenTests.fs
Comment (moodmosaic) [reply tests/Hedgehog.Tests/GenTests.fs]: Done. I didn't see any issues.

**Comment by moodmosaic** [comment]
Just forced-pushed. Thanks for the review. :+1:

**Comment by adam-becker** [comment]
I'll let @TysonMN weigh in before merging.

**Review by TysonMN — APPROVED** [review]
LGTM :)

**Pull request merged**
